xbmc/system.h

```diff
@@ -94,6 +94,7 @@
 #define HAS_WIN32_NETWORK
 #define HAS_IRSERVERSUITE
 #define HAS_FILESYSTEM_SMB
+#define HAVE_LIBBLURAY_BDJ
```

Comment:
bluray is only an optional dependency so this definition should set by cmake if bluray is found